std/uuid.d

```diff
@@ -1085,8 +1088,14 @@ unittest
     return u;
 }
 
+private @trusted
+ref T unsafeTrustingArrayElementReinterpret(T)(ref ubyte[16] arr, size_t i)
+{
+    return *cast(T*)(arr.ptr+i);
```

Comment:
Okey dokey. Trusted lambda it is then.

On Feb 9, 2015 7:51 PM, "H. S. Teoh" <notifications@github.com> wrote:
>
> In std/uuid.d:
>
> > @@ -1085,8 +1088,14 @@ unittest
> >      return u;
> >  }
> >
> > +private @trusted
> > +ref T unsafeTrustingArrayElementReinterpret(T)(ref ubyte[16] arr,
size_t i)
> > +{
> > +    return *cast(T*)(arr.ptr+i);
>
> Unfortunately, that's precisely the usage that Walter and Andrei strongly
disagree with. You'll have to take it up with them, since they have made it
clear that we are not to merge anything that abuses @trusted in this way.
>
> —
> Reply to this email directly or view it on GitHub.